store/rootmulti/store.go

```diff
@@ -586,7 +590,12 @@ func (rs *Store) PruneStores(clearPruningManager bool, pruningHeights []int64) (
 
 	rs.logger.Debug("pruning heights", "heights", pruningHeights)
 
-	for key, store := range rs.stores {
+	keys := make([]types.StoreKey, 0, len(rs.stores))
+	for key := range rs.stores {
+		keys = append(keys, key)
+	}
+	for _, key := range keys {
+		store := rs.stores[key]
```

Comment:
This code isn't sorting the keys, so AFAICT doesn't produce behavior any different than ranging over the map directly?